e2e/cypress/tests/integration/enterprise/extend_session/not_extended_when_disabled/with_ldap_login_spec.ts

```diff
@@ -10,15 +10,14 @@
 // Stage: @prod
 // Group: @enterprise @not_cloud @extend_session @ldap
 
-import ldapUsers from '../../../../fixtures/ldap_users.json';
-
 import {verifyExtendedSession, verifyNotExtendedSession} from './helpers';
 
 describe('Extended Session Length', () => {
     const sessionLengthInHours = 1;
     const setting = {
         ServiceSettings: {
             SessionLengthWebInHours: sessionLengthInHours,
+            ExtendSessionLengthWithActivity: false,
```

Comment:
Do we need to add this server setting? As we are just adding types in this PR?

Reply:
Actually the reason why I added this because type error
<img width="782" alt="Screen Shot 2565-11-04 at 20 50 49" src="https://user-images.githubusercontent.com/30198386/199988990-5488da97-e0d0-4fdf-b8f5-d8ed97dde3f8.png">